e2e/setup.ts

```diff
@@ -0,0 +1,49 @@
+import { chromium, firefox, webkit, type FullConfig } from '@playwright/test';
+import { CreateForkBody, createFork, forkRpcUrl } from './utils/tenderly';
+
+const forkConfig: CreateForkBody = {
+  network_id: '1',
+  alias: 'E2E-CI',
+};
+
+const browsers = { chromium, firefox, webkit };
+type BrowserName = keyof typeof browsers;
+
+async function globalSetup(config: FullConfig) {
+  console.log('Setting up Tenderly Fork');
+  console.time('Fork is setup');
+
+  // Create a Fork
+  const fork = await createFork(forkConfig);
+  process.env['TENDERLY_FORK_ID'] = fork.id;
+  const rpcUrl = forkRpcUrl(fork.id);
+  // Binance address
+  const imposter = '0x47ac0Fb4F2D84898e4D9E7b4DaB3C24507a6D503';
```

Comment:
should be env var and kept out of git log

Reply:
We discussed this in sync and couldn't find any reason to put it in env. As tests will rely on this address, so if each dev change it in his env, it might impact the tests.
Why do you think it should be in env ?

Reply:
as discussed - this is a very minor concern - resolving comment